lib/actions/api.js

```diff
@@ -559,6 +560,56 @@ export function findTrip(params) {
     }
   )
 }
+export function findGraphQLTrip(params) {
+  // Attempt an otp-2 graphQL request for the same trip before failing
+  // since the otp-2 rest api doesn't return the needed fields
```

Comment:
Convert this comment to JSDoc (insert blank line before JSDoc).

Reply:
Good call! Done in ca08debd338383fb327a3543108c92b3e269a740